content/docs/setup/kubernetes/prepare/platform-setup/openshift/index.md

```diff
@@ -39,3 +39,31 @@ constraints as part of sidecar injection:
 {{< text bash >}}
 $ oc adm policy add-scc-to-user privileged -z default -n <target-namespace>
 {{< /text >}}
+    
+## Automatic Injection
+Webhook and certificate signing requests support must be enabled for auto injection to work. To do so, you must modify the master configuration file for the cluster. 
+
+On the master node in the directory containing the master configuration file (likely /etc/origin/master/master-config.yaml), create a file named master-config.patch with the following contents:
```

Comment:
use an tip callout to indicate the default path.  See https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts for more details.

Reply:
Thanks. Updated.